serverless-functions/ReadSourceData/function.json

```diff
@@ -0,0 +1,19 @@
+{
+  "scriptFile": "__init__.py",
+  "bindings": [
+    {
+      "name": "blob",
+      "type": "blobTrigger",
+      "direction": "in",
+      "path": "source-data/{name}",
+      "connection": "PhiStorage"
+    },
+    {
+      "type": "serviceBus",
+      "direction": "out",
+      "connection": "AzureServiceBusConnectionString",
+      "name": "queue",
+      "queueName": "%ServiceBusQueueName%"
+    }
+  ]
+}
```

Comment:
This is interesting.  I assume this is how a serverless function is defined in Azure which uses the logic within the __init__.py file.  Is this correct?

Reply:
Yep! The `function.json` file is a configuration file for the function, the init file is the actual function code.